src/components/NavBar/NavMenu.jsx

```diff
@@ -75,7 +75,14 @@ const NavMenu = ({
       open={openMenu}
       onClose={handleMenuClose}
       onClick={handleMenuClose}
-      sx={{ mt: 5, backgroundColor: 'rgba(1, 121, 105, 0.2)' }}
+      slotProps={{
+        paper: {
+          style: {
+            marginTop: '10px'
+          }
+        }
+      }}
```

Comment:
I'm a bit curious about what this line does specific. Is this for updating the styling the underlying paper component from MUI?

Reply:
Yes, you’re correct :)